__tests__/app.test.js

```diff
@@ -1,3 +1,32 @@
-describe("Name of the group", () => {
-  test("should ", () => {});
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const allData = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(allData);
+});
+
+describe("GET", () => {

```

Comment:
Have a think about how we want to organise these tests (there's going to be a lot of them by the end).

Here it looks like you will be organising all GET requests together in one describe block.  There will be several GET requests, all of which need several different tests, which might end up being confusing if they're all in one describe block.  How could we re-organise/re-name these blocks to make it very clear what is happening.  Message me if you want to have a chat about it.

Reply:
I can put a describe block in another describe block? This way its separates each GET test